### ISSUE TITLE
Some metrics(disk,network) are extraordinay big.

### ISSUE THREAD

**Issue by mmuto118** [prometheus-pve/prometheus-pve-exporter#285]
I installed prometheus-pve-exporter in our proxmox.
I guess the below metrics are extraordinay big, I just looked into the issue but I have no clue about this.
```
pve_disk_read_bytes{id="qemu/xxxx"} 1.0617526039e+011 --> approximately 98.88 GB
pve_disk_write_bytes{id="qemu/xxxx"} 1.22488121344e+012 --> approximately 1140.76GB
pve_network_receive_bytes{id="qemu/xxxx"} 2.672781732581e+01  -->approximately 2489.22 GB
pve_network_transmit_bytes{id="qemu/100376"} 2.09707014379e+011 --> approximately 195.28 GB
```
Thanks in advance.

**Comment by mentos1386** [comment]
Hey, I assume the value is correct. If you read the "README.md" you might notice the description for this metric being "The amount of bytes the guest read from its block devices since the guest was started".

So it will grow until the "prometheus-pve-exporter" is restarted or quest is deleted.

What you might be interested is in using the `rate` function that prometheus provides. Like `rate(pve_network_receive_bytes{id="qemu/xxxx"}[5m])` to get the "network received per second" value.

To maintainers: The metric type should probably be `counter` instead of `gauge`. And the metric should have `_total` suffix. See https://prometheus.io/docs/tutorials/understanding_metric_types/.

**Comment by znerol** [comment]
> To maintainers: The metric type should probably be counter instead of gauge. And the metric should have _total suffix.

This is true. PR welcome. Add the counters and deprecate the gauges (we will not remove them for some time in order to avoid breaking existing deployments).
